galasa-parent/dev.galasa.framework.api.common/src/testFixtures/java/dev/galasa/framework/api/common/mocks/MockFramework.java

```diff
@@ -57,9 +57,9 @@ public boolean isInitialised() {
     @Override
     public @NotNull IConfigurationPropertyStoreService getConfigurationPropertyService(@NotNull String namespace)
             throws ConfigurationPropertyStoreException {
-        if ((!namespace.equals(this.cpsService.getNamespaceInput()))&&(!this.cpsService.getNamespaceInput().equals("empty"))){
-            throw new ConfigurationPropertyStoreException();
-        }
+            if(this.cpsService.namespaceInput.equalsIgnoreCase("error")){
```

Comment:
Might be better for the test which requires this behaviour just to subclass the mock framework and override this method to throw the error ?